README.md

```diff
@@ -83,6 +91,12 @@ console.log('data', await body.json())
 console.log('trailers', trailers)
 ```
 
+_Note: Once a mixin has been called, the `ReadableStream` the body represents cannot be reused, thus calling additional mixins on `.body`, e.g. `.body.json(); .body.text()` will result in an error `TypeError: unusable` being thrown and returned through the `Promise` rejection._
+
```

Comment:
```suggestion
_Note: Once a mixin has been called, the the body cannot be reused, thus calling additional mixins on `.body`, e.g. `.body.json(); .body.text()` will result in an error `TypeError: unusable` being thrown and returned through the `Promise` rejection._

```

Reply:
You all are welcome to finish this out the way you see best, or close this one. No offense taken either way. I feel that I've been super accommodating and the process has just gotten a bit too pedantic for my liking. 🍻